pandas/tests/scalar/test_nat.py

```diff
@@ -315,30 +310,31 @@ def test_overlap_public_nat_methods(klass, expected):
 
 
 @pytest.mark.parametrize(
-    "compare",
+    "klass",
     (
-        _get_overlap_public_nat_methods(Timestamp, True)
-        + _get_overlap_public_nat_methods(Timedelta, True)
+        Timestamp,
+        Timedelta
     ),
```

Comment:
If you combine those four lines to a single line with `(Timestamp, Timedelta),`, it conforms to black formatting and the linting error will go away.

```diff
-    (
-        Timestamp,
-        Timedelta
-    ),
+    (Timestamp, Timedelta),
```